config.py

```diff
@@ -1,17 +1,17 @@
 import os
 from pathlib import Path
 
-SERVICES_LINKS_EXAMPLE_PATH = Path("data/services_links_example.yml")
-SERVICES_LINKS_PATH = Path("data/services_links.yml")
+DATA_PATH_FOLDER = "./data"
```

Comment:
By converting this to Path, the file can be much simpler:

```python
DATA_PATH_FOLDER = Path("./data")

SERVICES_LINKS_PATH = DATA_PATH_FOLDER / "services_links.yml"
GITHUB_PR_LIST = DATA_PATH_FOLDER / "github_pr_list.json"
```
and so on.

Instances of Path can use `/` for joining paths.

Reply:
the `/` hack is awesome !!! thank you